src/templates/mkt_nav.html

```diff
@@ -15,6 +24,18 @@
     {{ nav_item(url('new'), _('New'), 'new') }}
     {{ nav_item(url('recommended'), _('Recommended'), 'recommended') }}
     {{ nav_item('#', _('Categories'), 'categories', 'categories') }}
+    <div class="mkt-nav-root--divider"></div>
```

Comment:
Okay, I don't think that'll work since ```<mkt-nav-root>``` is already a list element. And I think there's only one ```<mkt-nav-root>``` per nav.